tests/unit/via/views/error_test.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+from _pytest.mark import param
+from h_matchers import Any
+from mock import sentinel
```

Comment:
Oof, [Via 3 depends on the third-party `mock`?](https://github.com/hypothesis/via3/blob/838a70d67a93d81db749ae715c26f38cafd284c7/requirements-test.in#L4) It should be using the standard library's version of mock: `from unittest import mock`, and it _does_ actually do that in other files. That dependency should be removed

Reply:
I wonder when that happened. I've just been using what's around

Reply:
This has been fixed up now along with the weird import above.

Reply:
Looks like this was added by me, but I don't know why... anyway gone now.